runtime/runtime/src/verifier.rs

```diff
@@ -124,32 +124,43 @@ pub fn verify_and_charge_tx_ephemeral(
     transaction_cost: &TransactionCost,
     block_height: Option<BlockHeight>,
     current_protocol_version: ProtocolVersion,
+    ephemeral_state: Option<(Account, AccessKey)>,
```

Comment:
I wonder if it is will be cleaner to do the following instead:

```
cached_account: Option<Account>, cached_access_key: Option<AccessKey>,
```

This will make the function more generic in terms of if the callers have either of the cached state, they can use it.

Reply:
Tbh, I don't see uses for it anywhere else in the codebase. I'd avoid considering more cases for now

Reply:
The reason I suggested the change is because I think it will make the code cleaner and simpler to read not necessarily to support potential other callers.